### ISSUE TITLE
2 apps in 1 project?

### ISSUE THREAD

**Issue by SonyKar** [becoming/0-java-fundamentals#5]
[Here](https://github.com/becoming/0-java-fundamentals/blob/master/08-Contact-List.md) after seeing our future app model, you feel like level has sharply increased. And if you do your app after video, you see that half of the created classes are empty. 

So, ok, if we add there code, why we keep them in one project. To launch console app we create object of corresponding class, but if we do so the half of written the classes do not participate in the app work.

If we want to launch web app, we create object of corresponding class, but if we do so the half of the written classes do not participate in the app work.

So, why do not we divide this 2 apps in 2 different projects?

**Comment by rodislav** [comment]
Indeed, that part is unfinished and should be done. I'll work on it in version 2 of the curriculum.
